sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuColumnarToRowExec.scala

```diff
@@ -356,7 +366,9 @@ object GpuColumnarToRowExecParent {
         // This number includes the 1-bit validity per column, but doesn't include padding.
         // We are being conservative by only allowing 100M columns until we feel the need to
         // increase this number
-        output.length <= 100000000) {
+        output.length <= 100000000 &&
+        // Check if the current CUDA device supports accelerated transpose.
```

Comment:
Is accelerated transpose a hardware feature? A driver feature? Can you clarify what causes this to fail on Pascal?

Reply:
I've been chasing the problem for a while now. I think this comment belies that fact. 
I should mention that this is a workaround.

Reply:
The description has been modified to highlight that this is a workaround.